pyproject.toml

```diff
@@ -16,8 +16,8 @@ packages = [
 safe-ds-runner = "safeds_runner.main:main"
 
 [tool.poetry.dependencies]
-python = "^3.11,<3.13"
-safe-ds = ">=0.14,<0.17"
+python = "^3.11,<3.12"
```

Comment:
@lars-reimann Should an issue be created to revert this later?

Reply:
No, I track this for the Safe-DS library already.